scripts/create-docker-machine-and-run-it

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+#
+# Setup a docker machine for development
+set -eu
+
+readonly PROGNAME=$(basename "$0")
+readonly ARGS=("$@")
+# Docker settings
+MACHINE_NAME="devel"
+HUGO_VERSION=0.20.2
+NODE_VERSION=6.10.0
```

Comment:
> Could you be a bit clearer? Do you mean I put it in a function?

Container version using a fixed node version. This is also a fixed node version. Suggested approach is to create a config file to manage versions of both Hugo and container.